main.py

```diff
@@ -59,6 +63,7 @@ def distributed_main(config):
             cpus_per_task=(args.num_workers + 1),
             tasks_per_node=(args.num_gpus if args.distributed else 1),
             nodes=args.num_nodes,
+            slurm_constraint='volta32gb',
```

Comment:
I also found this constraint to not work when `--amp` is set to `False`. Any obvious reason that may be?

Reply:
This shouldn't be related to AMP.